Packs/ApiModules/Scripts/MicrosoftGraphMailApiModule/MicrosoftGraphMailApiModule.yml

```diff
@@ -13,7 +13,7 @@ system: true
 scripttarget: 0
 dependson: {}
 timeout: 0s
-dockerimage: demisto/crypto:1.0.0.52480
+dockerimage: demisto/crypto:1.0.0.96042
```

Comment:
Need also release note and to all the integration that depends on this api module (should happen automticly when running demisto-sdk update-release-notes)